bagel/tests/data/example_synthetic.jsonld

```diff
@@ -9,17 +9,16 @@
       "@id": "bg:hasContrastType"
     },
     "schemaKey": "@type",
-    "Assessment": "bg:Assessment",
     "Bagel": "bg:Bagel",
     "BaseModel": "bg:BaseModel",
+    "ControlledTerm": "bg:ControlledTerm",
```

Comment:
Do we actually need this in the context? Would "ControlledTerm" ever appear as an actual annotation?